evap/student/templates/student_vote.html

```diff
@@ -194,12 +197,52 @@ <h3 class="mb-3">{{ evaluation.full_name }} ({{ evaluation.course.semester.name
                     if (textResultsPublishConfirmation.bottomCard) {
                         updateTextResultsPublishConfirmation();
                     }
-                }
+                },
+                onSave: function () {
+                    const lastSaved = $('#last-saved');
+                    const timeNow = new Date();
+                    localStorage.setItem(lastSavedStorageKey, timeNow.toString());
+                },
             });
+
+            function updateLastSavedLabel() {
+                    const timeNow = new Date();
+                    const lastSavedLabel = $('#last-saved');
+                    const lastSavedDate = new Date(localStorage.getItem(lastSavedStorageKey))
```

Comment:
```suggestion
                    const lastSavedDate = new Date(localStorage.getItem(lastSavedStorageKey));
```
:paintbrush: Playing manual linter here: There are some semicolon missing (multiple occurences)

*****

Also, you should check for `localStorage.getItem(lastSavedStorageKey) !== null` here (see below)